builder/azure/arm/template_factory_test.TestVirtualMachineDeployment13.approved.json

```diff
@@ -130,7 +130,7 @@
               "listeners": [
                 {
                   "certificateUrl": "",
-                  "protocol": "https"
+                  "protocol": "Https"
```

Comment:
Weird casing here, is this voluntary?

Reply:
This is what the new SDK sends, they resolve the same but we do a file compare to check in the test that causes failures cause https != https, Azure interprets them the same